grasp_planner:
  antipodal_grasp_weight: 3
  non_antipodal_grasp_weight: 1
  antipodal_collision_weight: 0
  non_antipodal_collision_weight: -1
  sq_wrist_distance_weight: -900
  sq_prev_distance_weight: -1500
